docs/operator-guides/reset.md

```diff
@@ -24,13 +24,7 @@ You will also automatically be prompted to reset affected streams if you edit an
 Similarly to a sync job, a reset can be completed as successful, failed, or cancelled. To resolve a failed reset, you should manually drop the tables in the destination so that Airbyte can continue syncing accurately into the destination. 
 
 ## Reset behavior
-When a reset is successfully completed, all the records are deleted from your destination tables (and files, if using local JSON or local CSV as the destination).
-
-:::info
-If you are using destinations that are on the [Destinations v2](/release_notes/upgrading_to_destinations_v2.md) framework, only raw tables will be cleared of their data. Final tables will retain all records from the last sync. 
-:::
-
-A reset **DOES NOT** delete any destination tables when using a data warehouse, data lake, database. The schema is retained but will not contain any rows.
+When a reset is successfully completed, all the records are deleted from your destination tables (and files, if using local JSON or local CSV as the destination), and then the next sync will begin.
```

Comment:
Only records not tables, that correct?

Reply:
Maybe... add a note:
For certified destinations this is the behavior...
for community connectors that can be different

Reply:
I don't think it is different for any destination - a reset means that you end up loosing the content in the tables, and then the next sync brings them back.  File sources are a little different in that the "erase" step doesn't always happen, but sometimes you do end up with an empty file from the reset sync